src/Combatants.js

```diff
@@ -12,7 +12,12 @@ class Combatants extends Component {
   render() {
     const maxRows = this.props.config.maxCombatants
     const dataArray = Object.keys(this.props.data)
-    const battler = dataArray.slice(0, maxRows)
+	const regexp = /[a-zA-Z]+\s+[a-zA-Z]+/g;
+    const battler = dataArray.filter(player => (
```

Comment:
Just one little thing here: do you have prettier installed in your IDE? If not, I'll set a thing later to make this better indented once we push our files (ie. a post-commit hook).

Reply:
...and you've done it here. Reviews should begin with the last ones, heh!

Reply:
i actually fixed spacing in a later thing :P